kcp/controllers/assets/istio-test-resources.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: kcp-gateway
+  namespace: default
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - hosts:
+        - "*"
+      port:
+        name: http
+        number: 80
+        protocol: HTTP
+---
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: kcp-events
+  namespace: default
+  labels:
+    kcp-watcher-resource: "true"
```

Comment:
does this label have a significance? If not please remove to avoid confusion

Reply:
It doesn't have any. Deleted 👍